src/Data/Auth/PrimaryAuthController.ts

```diff
@@ -26,11 +32,25 @@ export class PrimaryKeycloakController implements KeycloakController {
   }
 
   private getConfig(): KeycloakConfig {
-    return (
-      this.externalConfig || {
+    if (this.externalConfig && this.externalConfig?.method === "oidc") {
```

Comment:
previously we were returning either keycloak config provided by config.js file or this default one, now config provided can vary as it can also mention database, to keep login the same way I had to add this additional check